code/modules/wod13/onyxcombat.dm

```diff
@@ -578,3 +551,83 @@
 				elysium_checks = 0
 				if(HAS_TRAIT(src, TRAIT_ELYSIUM))
 					REMOVE_TRAIT(src, TRAIT_ELYSIUM, "elysium")
+
+/mob/living/carbon/human
+	var/current_police_raid = FALSE //lock to prevent multiple handle_police_raid() procs from going at the same time
+
+/mob/living/carbon/human/proc/handle_police_raid() //hooo boy
```

Comment:
Could you move this proc to a new file, under code\modules\vtr13\mobs\human\handle_police_raid.dm?